packages/main/src/RangeSlider.hbs

```diff
@@ -37,7 +37,7 @@
 		aria-labelledby="{{_ariaLabelledByStartHandleRefs}}"
 		aria-disabled="{{_ariaDisabled}}"
 	>
-		<ui5-icon name="source-code" slider-icon></ui5-icon>
+		<ui5-icon name="direction-arrows" slider-icon></ui5-icon>
```

Comment:
Remember, you need to also import the icon explicitly in the JS file. Maybe we need to create a lint rule for this.

Reply:
Done